CleanUtils.py

```diff
@@ -27,6 +27,8 @@ def create_features_list(frame):
 
 
 def common_feature(df_series):
+    # take a look on is_nan, this can be replaced with something like:
+    # df_series.isna().count() / df_series.shape[0]
```

Comment:
your suggestion can tell me what the percentage of nan from each column, but in my function I need to understand the percentage of all the diffrents values in the column. From that to understand if the max is above 90% in order my later imputation will not suffer from large bias.